xml_converter/generators/proto_templates/node.proto

```diff
@@ -0,0 +1,274 @@
+syntax = "proto3";
+import "google/protobuf/any.proto";
+
+package Proto_Node;
+
+message Category {
+    bool default_visibility = 1;
+    string display_name = 2;
+    bool is_separator = 3;
+    string name = 4;
+    string tooltip_name = 5;
+    map <string, Category> children = 6;
+}
+
+message Icon {
```

Comment:
I think you are using nested message type definitions and thinking they are variable definitions.

Reply:
Reworked to avoid nested messages